site/schemas/roadwork.schema.ts

```diff
@@ -0,0 +1,88 @@
+import { z } from "zod";
+
+const workStatus = z.union([
+    z.literal("Works planned"),
+    z.literal("Works in progress"),
+    z.literal("Works completed"),
+    z.literal("Works cancelled"),
+    z.literal("Non notifiable works"),
+    z.literal("Unattributable works"),
+    z.literal("Historical works"),
+    z.literal("Section 81 works"),
+]);
+
+export type WorkStatus = z.infer<typeof workStatus>;
+
+export const worksLocationType = [
+    "Footway",
+    "Carriageway",
+    "Verge",
+    "Footpath",
+    "Parking place",
+    "Bus stop or stand",
+    "Cycle hire docking station",
```

Comment:
should we be using literals here too?

Reply:
This was a copied over from ref data service but after taking another look we're not actually using the worksLocation type so I'll remove it since its not needed.